db/src/main/kotlin/edu/artic/db/AppDataApi.kt

```diff
@@ -16,5 +16,21 @@ interface AppDataApi {
 
 
     @GET("appData-v2.json")
-    fun getBlob(@Header(Constants.DOWNLOAD_IDENTIFIER_HEADER) header: String): Observable<Result<ArticBlobData>>
+    fun getBlob(
+            @Header(Constants.DOWNLOAD_IDENTIFIER_HEADER) header: String
```

Comment:
`Constants` seems a little generic and god-object like. Make it more specific like `ApiConstants`